spec/convertion_by_format/check_open_docx_by_screen_spec.rb

```diff
@@ -17,12 +18,13 @@
       response = converter.perform_convert(url: uri.normalize.to_s, outputtype: 'png')
       expect(response[:url].nil?).to be_falsey
       expect(response[:url].empty?).to be_falsey
-      expect(ImageHelper.get_image_size(response[:url])).to be > StaticData::MIN_DOCX_IMAGE_SIZE
+      data_file = ImageHelper.get_image_size(response[:url])
```

Comment:
`image_size = ImageHelper.get_image_size...`

`data_file` is to general name and give no info